carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/DriverHelper.java

```diff
@@ -1120,8 +1120,12 @@ public boolean waitUntil(ExpectedCondition<?> condition, long timeout) {
 		        .ignoring(WebDriverException.class)
 				.ignoring(NoSuchSessionException.class);
 		try {
+		    long startMillis = System.currentTimeMillis();
 			wait.until(condition);
 			result = true;
+            if ((System.currentTimeMillis() - startMillis) > 2 * RETRY_TIME) {
+                LOGGER.debug("Your retry_interval is too low! Increase it or improve your hardware");
```

Comment:
let's use error as it is very important signal that current hardware is not enough